linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_maxclassrepeat/rule.yml

```diff
@@ -2,7 +2,7 @@ documentation_complete: true
 
 prodtype: rhel7,rhel8,fedora,ol7,ol8,rhv4
 
-title: 'Set Password to Maximum of Consecutive Repeating Characters from Same Character Class'
+title: 'Ensure PAM Enforces Password Requirements - Maximum of Consecutive Repeating Characters from Same Character Class'
```

Comment:
```suggestion
title: 'Ensure PAM Enforces Password Requirements - Maximum Consecutive Repeating Characters from Same Character Class'
```

Reply:
That is a interesting suggest and commit feature by GitHub.